src/Migrate.php

```diff
@@ -624,7 +625,17 @@ function (array $v) use ($database, $schemaName, $tableName) {
                     Helper::quoteIdentifier($view['schema_name'])
                 ));
 
-                $this->destinationConnection->query($view['text']);
+                try {
+                    $this->destinationConnection->query($view['text']);
+                } catch (Throwable $e) {
+                    $this->logger->info(sprintf(
+                        'Skip creating view %s.%s.%s. Error: "%s".',
```

Comment:
ahhh, but please keep `Warning:` text so we can gather them:)

Reply:
okok :)